DEVELOPMENT_ENVIRONMENT.md

```diff
@@ -95,5 +112,48 @@
 	- under "Eclipse Formatter config file", select ``sormas-base/java-formatter-profile.xml``
 	- check optimize imports and, for "Iport order", select ``sormas-base/java-importorder-profile.importorder``
 	- **Important:** select "Do not format other file types by IntelliJ formatter"
-	- go to Preferences -> Editor -> Code style -> Java : set class and static names counts for import with * to 99
+	- go to Settings -> Editor -> Code style -> Java : set class and static names counts for import with * to 99
 	- for Android Studio, code formatting is usually done with Ctrl+Alt+L. For automatic formatting, it's recommended to use the plugin Save Actions (https://plugins.jetbrains.com/plugin/7642-save-actions)
+
+
+## Issues which can appear during installation process of the project
+* (1)If was set once, sormas_user password for postgres DB to `password1`, at next run of .\server-setup.sh you need to use the same password.
+If you will use another password, that the DB connection will be refused. Because the preview password was already set for created DBs, users, etc.
+
+* (2)Payara version. As by today (18.12.2020) the latest payara version is 5.2020.7. Current version of the application cannot run with this version. Is needed to have maximum version of payara - 5.2020.2 (this is the maximum version on which application does not have problems and can run). Production PAYARA_VERSION is 5.194.
```

Comment:
Remove this. We fix the automatic setup which contains the Payara version to be used.